src/main/flight/failsafe.h

```diff
@@ -34,7 +36,9 @@ typedef struct failsafeConfig_s {
     uint8_t failsafe_kill_switch;           // failsafe switch action is 0: identical to rc link loss, 1: disarms instantly
     uint16_t failsafe_throttle_low_delay;   // Time throttle stick must have been below 'min_check' to "JustDisarm" instead of "full failsafe procedure".
     uint8_t failsafe_procedure;             // selected full failsafe procedure is 0: auto-landing, 1: Drop it
-} failsafeConfig_t;
+} PG_PACKED failsafeConfig_t;
```

Comment:
Be very carefull with PG_PACKED. `failsafe_throttle_low_delay` is unaligned - gcc will generate correct code when it is accessed through  `failsafeConfig_t`, but it will cause problems if you try to pass it around using pointer